galaxy/templates/configmap-nginx.yaml

```diff
@@ -55,7 +55,7 @@ data:
                 include uwsgi_params;
             }
 
-            location {{ template "galaxy.add_trailing_slash" .Values.ingress.path }}_x_accel_redirect {
+            location {{ template "galaxy.add_trailing_slash" .Values.ingress.path }}_x_accel_redirect/ {
```

Comment:
It's very strange that this trailing slash would be needed since usegalaxy.org doesn't seem to be using it, and it does work fine on the GVL, which makes me believe that something else may be going on. Still I can't see any harm in merging this, so will do so now pending more data.